### ISSUE TITLE
Limit the evaluated files

### ISSUE THREAD

**Issue by gian1200** [htmlhint/vscode-htmlhint#201]
Currently the extension evaluates all files that are open. Sometimes it evaluate files that should not be parsed (generated files, files under node_modules, target, etc.).

Is there a way to tell HTMLhint which files should only be evaluated?

CLI already have `ignore` option (https://htmlhint.com/docs/user-guide/usage/options), which AFAIK is currently not possible to use with the extension. However, it'd be nice to have also a "files" like option ("only files to evaluate"), similar to other linters (e.g. ESLint); ~if not under `.htmlhintrc`,~ maybe inside VSCode settings (settings.json).

**Comment by coliff** [comment]
I agree that this would be a helpful feature. The ignore option would ideally be added to the .`htmlhintrc` config and the VS Code would automatically read and use that.

**Comment by gian1200** [comment]
It shouldn't be inside `.htmlhintrc`, because it is only prepared to store rules. It'd break HTMLHint CLI (unless structure is changed by HTMLHint team first).

For the extension, it currently makes more sense to put it in `.vscode\settings.json`.

--------------

I just find out https://github.com/htmlhint/vscode-htmlhint/tree/main/htmlhint#settings, so it could be put there

```json
{
  "htmlhint.ignore": [
    "path1",
    "path2"
  ]
}
```

---------------

**Additional question**: could it also be defined like this?

```json
{
  "htmlhint": {
    "documentSelector": [
      "html",
      "htm",
      "twig"
    ],
    "ignore": [
      "path1",
      "path2"
    ]
    "options": {
      "tagname-lowercase": false,
      "attr-lowercase": true,
      "attr-value-double-quotes": true,
      "doctype-first": true
    }
  }
}
```

**Comment by gian1200** [comment]
PS: format (--format) can also be added

**Comment by coliff** [comment]
We just added the new 'Use .gitignore' feature! It's turned off by default but I hope it helps!

https://marketplace.visualstudio.com/items?itemName=HTMLHint.vscode-htmlhint#skipping-linting-for.gitignore-d-files

(I don't intend to add a more general option for specifying exactly which files to limit linting at this time).
